UseCase71/src/V1/HasDeprecatedFields.prefab.definition.yml

```diff
@@ -11,10 +11,27 @@ properties:
     data_type: bool
     record_key: deprecated_field
     nullable: true
-    deprecated: true
+    is_deprecated: true
     deprecated_message: 'in favor of replacement_field'
-    replacement: 'replacement_field'
+    replacement: replacement_field
   replacement_field:
     data_type: string
     record_key: deprecated_field
     nullable: true
+  deprecated_simply:
+    data_type: bool
+    record_key: deprecated_simply
+    nullable: true
+    is_deprecated: true
+  deprecated_without_replacement:
+    data_type: bool
+    record_key: deprecated_without_replacement
+    nullable: true
+    is_deprecated: true
+    deprecated_message: 'to prove this works'
+  deprecated_without_message:
+    data_type: bool
+    record_key: deprecated_without_reason
```

Comment:
Possibly intentional but the `record_key` doesn't match the field name

Reply:
Heh, not intentional. I changed one and forgot to change the other.